frontend/src/app/modules/home/dashboard/dashboard.component.sass

```diff
@@ -63,6 +63,4 @@
         flex-grow: 1
         flex-shrink: 0
         flex-basis: 350px
-        height: 419px
-
-
+        height: 492px
```

Comment:
Please use sass variables instead of pixel constants.

Reply:
Done.